files/validator.py

```diff
@@ -100,19 +91,17 @@ def _is_list(self, mapping_vars, mapping):
             return False
         return True
 
-    def _print_finish_error(self):
+    @staticmethod
+    def _print_finish_error():
```

Comment:
same for all other methods, as a side note, I am not sure we need a class object here at all, maybe worth adding a todo instead.

Reply:
Reverting all.